index.html

```diff
@@ -0,0 +1,214 @@
+                    <li><a href="#program" class="nav-link">Program</a></li>
+                    <li><a href="#speakers" class="nav-link">Speakers</a></li>
+                    <li><a href="#partners" class="nav-link">Partners</a></li>
+                    <li><a href="./about.html" class="nav-link about">About</a></li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+    <section id="home" class="hero">
+        <div class="hero-container">
+            <h3>You have been waiting for this</h3>
+            <h1 id="hero-title">Open Source Summit 2022</h1>
+            <div id="hero-text">
+                <p>Open Source Summit is the premier event for open source developers, technologists, and community
+                    leaders to collaborate, share information, solve problems, and gain knowledge, furthering open
+                    source innovation and ensuring a sustainable open source ecosystem. It is the gathering place for
+                    open-source code and community contributors.</p>
+            </div>
+            <div id="hero-info">
+                <h4>📆 June 21 - 24</h4>
+                <h4>🇺🇸 Austin, Texas</h4>
+                <h5>#OSSUMMIT</h5>
+            </div>
+        </div>
+    </section>
+
+    <section id="program">
+        <h3 class="section-title">Main Program</h3>
+        <div class="program-container">
+            <div class="program-card">
+                <div id="lecture"></div>
+                <h4 class="progam-title">Lecture</h4>
+                <p class="program-text">Access leading experts to learn how to navigate the complex open source
+                    environment</p>
+            </div>
+            <div class="program-card">
+                <div id="meet"></div>
+                <h4 class="progam-title">Meet</h4>
+                <p class="program-text">Learn about the latest trends in open source and open technologies</p>
+            </div>
+            <div class="program-card">
+                <div id="learn"></div>
+                <h4 class="progam-title">Learn</h4>
+                <p class="program-text">Gain a competitive advantage by learning about the latest in innovative open
+                    solutions </p>
+            </div>
+            <div class="program-card">
+                <div id="explore"></div>
+                <h4 class="progam-title">Explore</h4>
+                <p class="program-text">Explore career opportunities with the world’s leading technology companies</p>
+            </div>
+            <div class="program-card">
+                <div id="future"></div>
+                <h4 class="progam-title">Future</h4>
+                <p class="program-text">Find out what industry-leading companies and projects are doing in the future,
+                    and where technologies are headed</p>
+            </div>
+        </div>
+        <a class="program-btn" href="#">See The Whole program</a>
```

Comment:
- [ ] Kindly be noticed that the `See The Whole program` button should be under the cards as the design guideline shows.

Your Design | Design Guideline
---------------- | ------------------------
![Screenshot from 2022-06-23 19-12-38](https://user-images.githubusercontent.com/33752735/175356940-b8a034fa-ebb0-4ef5-a437-37299e3ecae0.png) | ![screenshot](https://user-images.githubusercontent.com/49064106/175345574-2c1773b6-c9a1-457e-864c-fdc0edc82605.png)